gui.py

```diff
@@ -1917,6 +1928,9 @@ def priority_mode(self, event: tk.Event[ttk.Combobox]) -> None:
                 self._settings.priority_mode = value
                 break
 
+    def unlinked_campaigns(self) -> None:
```

Comment:
The "Allow unlinked games" checkbox should probably be above the priority mode in the GUI.